lbuild/environment.py

```diff
@@ -203,6 +204,8 @@ def template(self, src, dest=None, substitutions=None, filters=None):
             else:
                 dest = src
 
+        src = self.repopath(src)
```

Comment:
> Btw, is it possible to print the module name from which the exception was thrown? 

Either add it to the exception text (e.g. `"In module '{}': ...", self.module.fullname`) or create a specialized exception for module build errors which includes the module as a parameter (similar to the forward exception). Then we just need a handling method to format that properly in the main.

> Otherwise good luck finding the problem.

"Something went wrong" :smile: